src/canCallNumberFunctionOn.js

```diff
@@ -0,0 +1,12 @@
+import { both, complement, hasIn, either, __ } from 'ramda';
```

Comment:
Let's drop this file. Files of `src/` directory should only form public APIs.

Instead let's create `src/internal/isCoercible.js` file and export the `isCoercible` function from there.

This function may become part of RA public API in future if there will be more usecases. We can implement it as follows:

```js
const hasToString = both(isObj, hasIn('toString'));
const hasValueOf = both(isObj, hasIn('valueOf'));
const isNotSymbol = complement(isSymbol);

const isCoercible = both(isNotSymbol, either(hasToString, hasValueOf));

export default isCoercible;
```